dali/operators/decoder/image_decoder.cc

```diff
@@ -1,4 +1,4 @@
-// Copyright (c) 2019, NVIDIA CORPORATION. All rights reserved.
+// Copyright (c) 2019-2021, NVIDIA CORPORATION & AFFILIATES. All rights reserved.
```

Comment:
This file is unchanged, except for the copyright. Revert?

Reply:
Done